app/src/main/scala/org/alephium/explorer/service/BlockFlowSyncService.scala

```diff
@@ -183,7 +184,22 @@ case object BlockFlowSyncService extends StrictLogging {
     for {
       localTs  <- getLocalMaxTimestamp()
       remoteTs <- getRemoteMaxTimestamp()
-    } yield fetchAndBuildTimeStampRange(step, backStep, localTs, remoteTs)
+    } yield {
+      TimeUtil.buildTimeStampRangeOrEmpty(
+        step     = step,
+        backStep = backStep,
+        localTs  = localTs,
+        remoteTs = remoteTs
```

Comment:
maybe when arguments are the same as the parameters, we could write them once without the `=`?